zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/config/OpenApiConfig.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.zeebe.gateway.rest.config;
+
+import io.swagger.v3.oas.models.ExternalDocumentation;
+import io.swagger.v3.oas.models.info.Info;
+import io.swagger.v3.oas.models.info.License;
+import org.springdoc.core.models.GroupedOpenApi;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class OpenApiConfig {
+
+  @Bean
+  public GroupedOpenApi zeebeApi() {
```

Comment:
🔧 I guess we should rather call this `camundaApi`? What about the Zeebe REST API, will this also be picked up, by the way? It's residing in https://github.com/camunda/camunda/blob/main/zeebe/gateway-protocol/src/main/proto/rest-api-v1.yaml.

Reply:
Just worked over this one to rename the method. 

Also the Group is filtering only for v2 apis on Camunda API

Should we create a group for V1? (as a follow-up)

<img width="1478" alt="image" src="https://github.com/user-attachments/assets/fc8dbde3-2f36-4c65-8653-1a9313c1a6ba" />

Reply:
> Should we create a group for V1? (as a follow-up)

That would be good, yeah